### PR TITLE
Support Symfony 5 for 1.x

### PR DESCRIPTION
**Pull request by fsecnd**

### CODE REVIEW

**Review by kroshp — APPROVED** [review]

File: src/EventListener/ControllerListener.php [hunk 1]
```diff
@@ -37,8 +41,6 @@ public function __construct(ConfigManager $configManager, ControllerResolverInte
 
     /**
      * Exchange default admin controller by custom entity admin controller.
-     *
-     * @param FilterControllerEvent $event
      */
     public function onKernelController(FilterControllerEvent $event)
```
Comment (yceruto) [inline src/EventListener/ControllerListener.php]: `ControllerEvent` should be used instead

File: src/EventListener/ExceptionListener.php [hunk 2]
```diff
@@ -31,25 +34,26 @@
...
         $this->twig = $twig;
         $this->easyAdminConfig = $easyAdminConfig;
 
         parent::__construct($controller, $logger);
     }
 
-    /**
-     * @param GetResponseForExceptionEvent $event
-     */
     public function onKernelException(GetResponseForExceptionEvent $event)
```
Comment (yceruto) [inline src/EventListener/ExceptionListener.php]: `ExceptionEvent` should be used instead

File: src/EventListener/ExceptionListener.php [hunk 3]
```diff
@@ -68,8 +72,6 @@ public function onKernelException(GetResponseForExceptionEvent $event)
     }
 
     /**
-     * @param FlattenException $exception
-     *
      * @return Response
      */
     public function showExceptionPageAction(FlattenException $exception)
```
Comment (yceruto) [inline src/EventListener/ExceptionListener.php:68]: please check this method, it has some PHP warnings to fix

**Review by yceruto — CHANGES_REQUESTED** [review]

**Comment by yceruto** [comment]
@fsecnd could you create a PR (like this one https://bitbucket.org/cocolabssas/cocorico/pull-requests/1690) to test the SBO before merging this? thanks!

**Review by yceruto — APPROVED** [review]

**Pull request merged**